docs/getting-started.md

```diff
@@ -26,7 +26,7 @@ Let's take a look at how to get started with `zrok`.
 
 ## Downloading zrok
 
-Releases are also available from the `zrok` project repository on GitHub at https://github.com/openziti/zrok/releases/latest. If you're a Linux user and you're not sure which download to use then check out the hints on [the Downloads page](./downloads.md).
+Releases are also available from the `zrok` project repository on GitHub at https://github.com/openziti/zrok/releases/latest. If you're a Linux user check out [the Linux Service guide](/guides/linux-service.md).
```

Comment:
I think we need to be careful with this language... There are linux users who want to use "zrok Edge" (or "frontdoor", or whatever we end up calling it).. who want to install zrok as a service. And there are also linux users who just want to use the CLI like any other user. And this text will confuse the non-service users.